test/source/ErrorAggregatorTests.cpp

```diff
@@ -1,5 +1,7 @@
 #include <ErrorAggregatorTests.hpp>
 #include <testing_utils/DeviceIdentificationHelper.h>
+#include <bringauto/modules/ModuleManagerLibraryHandlerLocal.hpp>
```

Comment:
I cannot find any related tests. Can we add unit tests for updates and verify that the updates do not "break" tests in a manner "It works, but it shall not"?